### PR TITLE
[system_core] Add AgentCheckTest mocked test

### PR DESCRIPTION
**Pull request by degemer**
\+ as usual pep8/flake8 cleanup.

### CODE REVIEW

File: tests/test_system_core.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+class SystemCoreTestCase(AgentCheckTest):
+
+    CHECK_NAME = 'system_core'
+
+    def __init__(self, *args, **kwargs):
+        AgentCheckTest.__init__(self, *args, **kwargs)
+        self.config = {"instances": [{}]}
+
+    @mock.patch('psutil.cpu_times', return_value=MOCK_PSUTIL_CPU_TIMES)
+    def test_couch(self, mock_cpu_times):
```
Comment (LeoCavaille) [inline tests/test_system_core.py]: `couch` ?

File: tests/test_system_core.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ]
+
+
+class SystemCoreTestCase(AgentCheckTest):
+
+    CHECK_NAME = 'system_core'
+
+    def __init__(self, *args, **kwargs):
+        AgentCheckTest.__init__(self, *args, **kwargs)
+        self.config = {"instances": [{}]}
```
Comment (LeoCavaille) [inline tests/test_system_core.py]: nitpick: probably don't need to override `__init__`  and set an empty config as an instance attr?

File: util.py [hunk 3]
```diff
@@ -568,7 +568,7 @@ def is_darwin(name=None):
 
     @staticmethod
     def is_mac(name=None):
-        Platform.is_darwin(name)
+        return Platform.is_darwin(name)
```
Comment (LeoCavaille) [inline util.py]: Nice catch!

**Comment by degemer** [comment]
Thanks @LeoCavaille, updated as per your comments.

**Comment by LeoCavaille** [comment]
Awesome :+1:

**Pull request merged**